### PR TITLE
[tm] Chp3-2 Update layer symbology chapter

### PR DESCRIPTION
**Pull request by DelazJ**
While most of the text changes here are formatting matters (except an exercise added to address the new "protected_areas" layer), new data and new layers require screenshots updates.

### CODE REVIEW

File: source/docs/training_manual/answers/answers.rst [hunk 1]
```diff
@@ -84,15 +84,15 @@ There should be five layers on your map:
 
 * Verify that the colors are changing as you expect them to change.
 * It is enough to select the :guilabel:`water` layer in the legend and then click
-  on the |symbology|:sup:`Open the Layer Styling panel` button. Change the color
+  on the |symbology| :sup:`Open the Layer Styling panel` button. Change the color
   with the one you want and best fits the water layer.
```
Comment (havatv) [inline source/docs/training_manual/answers/answers.rst]:
```suggestion
  to one that fits the water layer.
```

File: source/docs/training_manual/basic_map/symbology.rst [hunk 2]
```diff
@@ -29,17 +29,19 @@ any vector layer.
...
+#. Right-click on the :guilabel:`landuse` layer in the layers list.
 #. Select the menu item :guilabel:`Properties...` in the menu that appears.
 
    .. note:: By default, you can also access a layer's properties by
      double-clicking on the layer in the Layers list.
 
-   .. tip:: The button |symbology| in the upper part of the Layer Panel will open
-     the Layer Styling Panel. You can use this panel to change the symbology of
-     the layer: changes will be applied immediately!
+   .. tip:: The button |symbology| in the upper part of the :guilabel:`Layers`
```
Comment (havatv) [inline source/docs/training_manual/basic_map/symbology.rst]:
```suggestion
   .. tip:: The button |symbology| in the upper left part of the :guilabel:`Layer Properties`
```

File: source/docs/training_manual/basic_map/symbology.rst [hunk 3]
```diff
@@ -29,17 +29,19 @@ any vector layer.
...
+     panel will open the :guilabel:`Layer Styling` panel. You can also use this
```
Comment (havatv) [inline source/docs/training_manual/basic_map/symbology.rst]:
```suggestion
     window will open the :guilabel:`Symbology` panel. You can also use this
```

Comment (havatv) [inline source/docs/training_manual/basic_map/symbology.rst]:
```suggestion
     panel will open the :guilabel:`Layer Styling` panel. You can use this
```

File: source/docs/training_manual/basic_map/symbology.rst [hunk 4]
```diff
@@ -56,8 +58,8 @@ begin by changing the color of the :guilabel:`landuse` layer.
 |basic| |TY|
 -------------------------------------------------------------------------------
 
-Change the :guilabel:`water` layer to a light blue color. Try to use the Layer
-Styling Panel instead of the :guilabel:`Properties` layer menu.
+Change the :guilabel:`water` layer to a light blue color. Try to use the
```
Comment (havatv) [inline source/docs/training_manual/basic_map/symbology.rst]:
```suggestion
Change the color of the :guilabel:`water` layer to light blue. Try to use the
```

File: source/docs/training_manual/basic_map/symbology.rst [hunk 5]
```diff
@@ -72,18 +74,18 @@ areas so as to make the map less visually cluttered.
...
    layer.
 
-   Under the :guilabel:`Symbology` tab, you will see the same kind of dialog as
-   before. This time, however, you're doing more than just quickly changing the
-   color.
-#. in the :guilabel:`Symbol layers` panel, expand the :guilabel:`Fill` dropdown
+   Under the |symbology| :guilabel:`Symbology` tab, you will see the same kind
+   of dialog as before. This time, however, you're doing more than just quickly
+   changing the color.
+#. In the symbol layers tree, expand the :guilabel:`Fill` dropdown
```
Comment (havatv) [inline source/docs/training_manual/basic_map/symbology.rst]:
```suggestion
#. In :guilabel:`Symbol layer type` dropdown,
```

File: source/docs/training_manual/basic_map/symbology.rst [hunk 6]
```diff
@@ -72,18 +74,18 @@ areas so as to make the map less visually cluttered.
...
    (if necessary) and select the :guilabel:`Simple fill` option:
```
Comment (havatv) [inline source/docs/training_manual/basic_map/symbology.rst]:
```suggestion
   select the :guilabel:`Simple fill` option:
```

File: source/docs/training_manual/basic_map/symbology.rst [hunk 7]
```diff
@@ -123,14 +127,14 @@ To enable scale-based rendering:
 
 #. Open the :guilabel:`Layer Properties` dialog for the :guilabel:`buildings`
    layer.
-#. Activate the :guilabel:`Rendering` tab.
+#. Activate the |rendering| :guilabel:`Rendering` tab.
 #. Enable scale-based rendering by clicking on the checkbox labeled
    :guilabel:`Scale dependent visibility`:
+#. Change the :guilabel:`Minimum` value to ``1:10,000``.
```
Comment (havatv) [inline source/docs/training_manual/basic_map/symbology.rst]:
```suggestion
#. Change the :guilabel:`Minimum` value to ``1:10000``.
```

File: source/docs/training_manual/basic_map/symbology.rst [hunk 8]
```diff
@@ -197,15 +200,17 @@ your layer that way.
...
 It's fun! But it probably has too many colors to use in a real map...
 
 .. _backlink-symbology-layers-1:
 
 |moderate| |TY|
 -------------------------------------------------------------------------------
 
-* Remembering to zoom in if necessary, create a simple, but not distracting
-  texture for the :guilabel:`buildings` layer using the methods above.
+Remembering to zoom in if necessary, create a simple, but not distracting
```
Comment (havatv) [inline source/docs/training_manual/basic_map/symbology.rst]:
```suggestion
Remembering to zoom in if necessary, and then create a simple, but not distracting
```

File: source/docs/training_manual/basic_map/symbology.rst [hunk 9]
```diff
@@ -149,19 +153,18 @@ to create more complex symbology. QGIS allows you to do this using symbol
 layers.
 
 #. Go back to the :guilabel:`landuse` layer's symbol properties panel (by clicking
-   :guilabel:`Simple fill` in the :guilabel:`Symbol layers` panel).
-
+   :guilabel:`Simple fill` in the symbol layers tree).
```
Comment (havatv) [inline source/docs/training_manual/basic_map/symbology.rst]:
```suggestion
   :guilabel:`Simple fill` in the :guilabel:`Symbol layer type` dropdown).
```

File: source/docs/training_manual/basic_map/symbology.rst [hunk 10]
```diff
@@ -149,19 +153,18 @@ to create more complex symbology. QGIS allows you to do this using symbol
...
    :guilabel:`No Pen` border style).
 
-#. Select the :guilabel:`Fill` in the :guilabel:`Symbol layers` panel. Then click
-   the :guilabel:`Add symbol layer` button:
-
    .. image:: img/add_symbol_layer_button.png
       :align: center
 
-#. Click on it and the dialog will change to look somewhat like this:
+#. Select now the :guilabel:`Fill` level in the tree and click
```
Comment (havatv) [inline source/docs/training_manual/basic_map/symbology.rst]:
```suggestion
#. Select the :guilabel:`Fill` level in the tree and click
```

File: source/docs/training_manual/basic_map/symbology.rst [hunk 11]
```diff
@@ -94,12 +96,14 @@ Now the :guilabel:`landuse` layer won't have any lines between areas.
...
 
-* Change the :guilabel:`water` layer's symbology again so that it is has a
+* Change the :guilabel:`water` layer's symbology again so that it has a
   darker blue outline.
 * Change the :guilabel:`rivers` layer's symbology to a sensible representation
   of waterways.
 
-Remember: you can use the |symbology| button and see instantly all the changes.
+Remember: you can use the |symbology| :sup:`Open the Layer Styling panel`
+button and see instantly all the changes. Also that panel allows you to undo
```
Comment (tomchadwin) [inline source/docs/training_manual/basic_map/symbology.rst]:
```suggestion
button and see all the changes instantly. That panel also allows you to undo
```

File: source/docs/training_manual/basic_map/symbology.rst [hunk 12]
```diff
@@ -149,19 +153,18 @@ to create more complex symbology. QGIS allows you to do this using symbol
...
-#. Select the :guilabel:`Fill` in the :guilabel:`Symbol layers` panel. Then click
-   the :guilabel:`Add symbol layer` button:
-
    .. image:: img/add_symbol_layer_button.png
       :align: center
 
-#. Click on it and the dialog will change to look somewhat like this:
+#. Select now the :guilabel:`Fill` level in the tree and click
+   the |signPlus| :sup:`Add symbol layer` button.
+   The dialog will change to look somewhat like this, with a new symbol layer
```
Comment (tomchadwin) [inline source/docs/training_manual/basic_map/symbology.rst]:
```suggestion
   The dialog will change to look something like this, with a new symbol layer
```

File: source/docs/training_manual/basic_map/symbology.rst (hunk 8 above)
Comment (tomchadwin) [reply source/docs/training_manual/basic_map/symbology.rst]: If this suggestion is accepted, I guess we should change "Remembering" to "Remember".

File: source/docs/training_manual/basic_map/symbology.rst [hunk 13]
```diff
@@ -247,9 +254,9 @@ To change the order of the symbol layers:
    .. image:: img/symbol_levels_dialog.png
       :align: center
 
-#. Select :guilabel:`Enable symbol levels`. You can then set the layer ordering
-   of each symbol by entering the corresponding level number. 0 is the bottom
-   layer.
+#. Check |checkbox| :guilabel:`Enable symbol levels`. You can then set the
+   layer ordering of each symbol by entering the corresponding level number.
```
Comment (tomchadwin) [inline source/docs/training_manual/basic_map/symbology.rst]:
```suggestion
   layer order of each symbol by entering the corresponding level number.
```

File: source/docs/training_manual/basic_map/symbology.rst [hunk 14]
```diff
@@ -265,13 +272,18 @@ To change the order of the symbol layers:
...
+|
+
 When you're done, remember to save the symbol itself so as not to lose your
 work if you change the symbol again in the future. You can save your current
-symbol style by clicking the :guilabel:`Save Style...` button under the
-:guilabel:`Symbology` tab of the :guilabel:`Layer Properties` dialog. Generally,
-you should save as :guilabel:`QGIS Layer Style File`.
+symbol style by clicking the :guilabel:`Save Style...` button at the bottom
+of the :guilabel:`Layer Properties` dialog. Generally,
+you can save as :guilabel:`QGIS QML Style File`.
```
Comment (tomchadwin) [inline source/docs/training_manual/basic_map/symbology.rst]: "Generally, you can" doesn't really make sense. Perhaps a brief reason for generally saving as QML would help?

File: source/docs/training_manual/basic_map/symbology.rst [hunk 15]
```diff
@@ -265,13 +272,18 @@ To change the order of the symbol layers:
...
+
+.. TODO: imho there's a mix of saving symbols and saving styles here that could
+ be misleading. Saving style should deserve its own section.
 
-Save your style under :file:`exercise_data/styles`. You can load a
+Save your style under :file:`solution/styles/better_roads.qml` folder. You can load a
```
Comment (tomchadwin) [inline source/docs/training_manual/basic_map/symbology.rst]:
```suggestion
Save your style in the :file:`solution/styles/better_roads.qml` folder. You can load a
```

File: source/docs/training_manual/basic_map/symbology.rst [hunk 16]
```diff
@@ -281,14 +293,14 @@ replacing will be lost.
 |moderate| |TY|
 -------------------------------------------------------------------------------
 
-* Change the appearance of the :guilabel:`roads` layer again.
+Change the appearance of the :guilabel:`roads` layer again.
 
-  The roads must be narrow and yellow, with a thin, pale gray outline and a
-  thin black line in the middle. Remember that you may need to change the layer
-  rendering order via the :menuselection:`Advanced --> Symbol levels...` dialog.
+The roads must be narrow and yellow, with a thin, pale gray outline and a
```
[the remark this answers is not in the document]
Comment (tomchadwin) [reply source/docs/training_manual/basic_map/symbology.rst]: How about "Make the roads narrow..."?

File: source/docs/training_manual/basic_map/symbology.rst [hunk 17]
```diff
@@ -412,6 +428,22 @@ benefit that you can change the size, shape and distance of the individual dots
 that make up the texture.
 
 
+.. _backlink-symbology-marker-line:
+
+|moderate| |TY|
+-------------------------------------------------------------------------------
+
+Apply to the :guilabel:`protected_areas` layer a green transparent fill color
```
[the remark this answers is not in the document]
Comment (tomchadwin) [reply source/docs/training_manual/basic_map/symbology.rst]: How about "Apply a green transparent fill color to the protected_areas layer"?

File: source/docs/training_manual/basic_map/symbology.rst [hunk 18]
```diff
@@ -412,6 +428,22 @@ benefit that you can change the size, shape and distance of the individual dots
...
+with outline looking like this:
```
[the remark this answers is not in the document]
Comment (tomchadwin) [reply source/docs/training_manual/basic_map/symbology.rst]:
```suggestion
and change the outline to look like this:
```

File: source/docs/training_manual/basic_map/symbology.rst [hunk 19]
```diff
@@ -535,10 +567,11 @@ The final result should look like this:
 In QGIS:
 
 #. Open the :guilabel:`Layer Properties` for the :file:`landuse` layer.
-#. In the :guilabel:`Symbology` tab, change the symbol structure by selecting
-   :guilabel:`SVG Fill` as :guilabel:`Symbol Layer Type` option, as shown below.
-#. Click the |browseButton| :guilabel:`Browse` button and then on :guilabel:`Select File...`
-   to select your SVG image.
+#. In the |symbology| :guilabel:`Symbology` tab, change the symbol structure by
+   selecting :guilabel:`SVG Fill` as :guilabel:`Symbol Layer Type` option, as
```
[the remark this answers is not in the document]
Comment (tomchadwin) [reply source/docs/training_manual/basic_map/symbology.rst]:
```suggestion
   changing the :guilabel:`Symbol Layer Type` to :guilabel:`SVG Fill````

File: source/docs/training_manual/basic_map/symbology.rst [hunk 20]
```diff
@@ -535,10 +567,11 @@ The final result should look like this:
...
+   shown below.
+#. Click the :guilabel:`...` button and then on
```
Comment (tomchadwin) [inline source/docs/training_manual/basic_map/symbology.rst]:
```suggestion
#. Click the :guilabel:`...` button and then
```

File: source/docs/training_manual/basic_map/symbology.rst [hunk 21]
```diff
@@ -265,13 +272,18 @@ To change the order of the symbol layers:
...
 
+|
+
 When you're done, remember to save the symbol itself so as not to lose your
 work if you change the symbol again in the future. You can save your current
-symbol style by clicking the :guilabel:`Save Style...` button under the
-:guilabel:`Symbology` tab of the :guilabel:`Layer Properties` dialog. Generally,
-you should save as :guilabel:`QGIS Layer Style File`.
+symbol style by clicking the :guilabel:`Save Style...` button at the bottom
+of the :guilabel:`Layer Properties` dialog. Generally,
```
Comment (DelazJ) [inline source/docs/training_manual/basic_map/symbology.rst]:
```suggestion
of the :guilabel:`Layer Properties` dialog.
```

File: source/docs/training_manual/basic_map/symbology.rst (hunk 14 above)
Comment (DelazJ) [reply source/docs/training_manual/basic_map/symbology.rst]:
```suggestion
We will be using the :guilabel:`QGIS QML Style File` format.
```

**Pull request merged**